Userland/Applications/Presenter/PresenterWidget.cpp

```diff
@@ -68,6 +72,9 @@ ErrorOr<void> PresenterWidget::initialize_menubar()
     m_next_slide_action = next_slide_action;
     m_previous_slide_action = previous_slide_action;
 
+    auto export_slides_action = GUI::Action::create("&Export Slides...", { KeyModifier::Mod_Ctrl, KeyCode::Key_E }, [this](auto&) { this->on_export_slides_action(); });
+    TRY(presentation_menu.try_add_action(export_slides_action));
```

Comment:
i don't think that this menu is a good place for it – all other actions in the menu seem to be related to navigation and display. Could we move it to the File menu instead?

Reply:
I noticed that too when sending the screenshot, but forgot to actually fix it